tests/integration/components/description-test.js

```diff
@@ -6,19 +6,28 @@ import { hbs } from 'ember-cli-htmlbars';
 module('Integration | Component | description', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('description content renders', async function (assert) {
+  test("old description content doesn't renders", async function (assert) {
```

Comment:
Small typo in `renders` -> `render`

Reply:
https://github.com/Real-Dev-Squad/website-www/pull/773#issuecomment-1863657827